applications/vessel-history/public/locales/en/translations.json

```diff
@@ -0,0 +1,57 @@
+{
+  "common": {
+    "activity": "activity",
+    "commingSoon": "coming soon!",
+    "currentTimeRange": "current time range",
+    "from": "from",
+    "info": "info",
+    "map": "map",
+    "offlineAccess": "offline access",
+    "since": "since",
+    "timeRange": "time range",
+    "to": "to",
+    "historyFor": "history for",
+    "previous": "previous"
+  },
+  "search": {
+    "description": "Search by vessel name or identification code (IMO, MMSI, VMS ID, etc…). You can narrow your search pressing the filter icon in the top bar",
+    "noResults": "Can't find the vessel you are looking for? Try using MMSI, IMO or Call Sign",
+    "notAllowed": "Search not allowed"
+  },
+  "selects": {
+    "allSelected": "All",
+    "selected": "selected"
+  },
+  "vessel": {
+    "authorizationPlural": "authorizations",
+    "historyLabelByField": "{{label}} $t(common.historyFor) {{vesselName}}",
+    "noAuthorizations": "No authorizations found",
+    "plusPreviousValuesByField": "+{{quantity}} $t(common.previous) {{fieldLabel}}",
```

Comment:
you need to be so sure about this, I mean use parts of another translations, when this become big and you don't remember all the relations (also if you remember) and you need to modify the "common.previous" text maybe in only one language and the new word doesn't match in the places where is used

Reply:
Yeah, makes sense. I'll change it